app/presenters/tree_node/node.rb

```diff
@@ -54,7 +54,7 @@ def expand
     end
 
     def hide_checkbox
```

Comment:
@skateman does it actually make sense to keep this method? Where do we use it, except here?

Reply:
Yes, a few lines lower...all the methods should be there for compatibility